.github/workflows/rust.yml

```diff
@@ -93,6 +93,11 @@ jobs:
       - name: Pin cc if rust 1.29
         if: matrix.rust == '1.29.0'
         run: cargo generate-lockfile --verbose && cargo update -p cc --precise "1.0.41" --verbose
+      - name: rustfmt on stable
+        if: matrix.rust == 'stable'
+        env:
+          DO_LINT: true
+        run: cargo generate-lockfile --verbose && cargo update -p cc --precise "1.0.41" --verbose
```

Comment:
I guess you meant to call `test.sh` here?

Reply:
Yes, indeed :facepalm:. Sorry about that. No more 3am coding more me.